### PR TITLE
NC-584 reported bug.

### PR DESCRIPTION
**Pull request by Joeper214**
ref: https://skyarch.backlog.jp/view/NC-584

Please review and merge PR. @root-beer. :bow:

### CODE REVIEW

File: app/views/vue/_elb_tabpane.html.erb [hunk 1]
```diff
@@ -90,7 +90,7 @@
   <div class="form-actions-top col-xs-12" v-if="unregistereds.length !== 0">
     <div class="input-group col-xs-12 col-sm-5">
       <select class="form-control input-sm" v-model="selected_ec2">
-        <option>Select!</option>
+        <option></option>
         <option v-for="x in unregistereds" :value="x.physical_id">{{* x.screen_name}}</option>
```
Comment (root-beer) [inline app/views/vue/_elb_tabpane.html.erb]: I think it's better to indicate both of screen_name and physical_id.

Comment (Joeper214) [inline app/views/vue/_elb_tabpane.html.erb]: :+1:

Comment (Joeper214) [inline app/views/vue/_elb_tabpane.html.erb]: Like this?  @root-beer 
like

Comment (root-beer) [inline app/views/vue/_elb_tabpane.html.erb]: :+1:

**Comment by root-beer** [comment]
Merged it. Thanks! :bow:

**Pull request merged**